onboarding/src/Components/Employee/EmployeeFormPages/EmployeeFormPagesList.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const EmployeeFormPagesList = ({ pagesList, switchPage }) => {
+  const moveToForm = (page) => {
+    switchPage(page);
+  };
+
+  const pages = pagesList.map((page) => (
+    <tr key={page.id}>
+      <td>
+        <a href="#" onClick={(e) => moveToForm(page)}>
```

Comment:
(*)
`e.preventDefault()`

Reply:
I didn't think that going to the top of page was expected;
but if it is then we can skip that.